### PR TITLE
Graph Top N States

### PR DESCRIPTION
**Pull request by Phani9231**

### CODE REVIEW

**Comment by nallapuneniprudhvi** [comment]
In graphTonNStates function you should select **title** dynamically but instead you gave it as **Top n featured**

**Comment by nallapuneniprudhvi** [comment]
In graphTonNStates you should select **title** given in the function but instead you gave it as **Top n featured**

File: hw6_social.py [hunk 1]
```diff
@@ -181,7 +301,14 @@ def graphStateCounts(stateCounts, title):
...
 Returns: None
 '''
+import heapq
 def graphTopNStates(stateCounts, stateFeatureCounts, n, title):
+    featurerate={}
+    topstates={}
+    for i in stateFeatureCounts:
+        featurerate[i]=(stateFeatureCounts[i]/stateCounts[i])
+    topstates=dict(Counter(featurerate).most_common(n))
+    graphStateCounts(topstates, "Top n Featured")
```
Comment (nallapuneniprudhvi) [inline hw6_social.py]: In graphTonNStates you should select **title** given in the function but instead you gave it as **Top n featured**

Comment (Phani9231) [reply hw6_social.py]: corrected it

**Comment by nallapuneniprudhvi** [comment]
Code is working well without any errors

**Pull request merged**